app.rambox.ramboxce.metainfo.xml

```diff
@@ -0,0 +1,28 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>app.rambox.ramboxce</id>
+  <metadata_license>GPL</metadata_license>
```

Comment:
`CC0-1.0` is the prefered license for the metadata.
GPL (any version) is not acceptable.